src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,137 @@
 package core.basesyntax;
 
+import java.util.Map;
```

Comment:
The import statement for `Map` is unnecessary unless you are using `Map` elsewhere in your code. Consider removing it if it's not needed.